restructure quota mgmt docs v3 (#5141)
getsentry/sentry-docs: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

Applying the proper filters, SDK configuration, and rate limits is an iterative and on-going process, but these tips will show you how to get the most out of your error events.

## 1. Spike Protection

> Sentry also applies a dynamic rate limit to your account designed to protect you from short-term spikes. **However, we still recommend applying all of the previously mentioned methods.**

Author: ```suggestion
Sentry also applies a dynamic rate limit to your account designed to protect you from short-term spikes. **However, we still recommend applying all of the other quota management methods that make sense for your use case.** Learn more in [Spike Protection](/product/accounts/quotas/#1-spike-protection).
```

## 2. Event Usage Stats {#common-workflows-for-managing-your-event-stream}

Once you've completed the steps above, you can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer some key questions like:

- What's the breakdowndown of my incoming events?
- Which projects are consuming my quota?
- Which issues are consuming my quota?

> The answers to these questions can help you figure out where you need to do further fine-tuning of your filters, limits, or SDK configuration.

Author: ```suggestion
The answers to these questions can help you figure out where you need to do further fine-tuning of your filters, limits, or SDK configuration. Learn more in [Event Usage Stats](/product/accounts/quotas/#common-workflows-for-managing-your-event-stream).
```

## 3. Quota Availability

> blah blah

Author: ```suggestion
blah blah

Learn more in [Quota Availability](/product/accounts/quotas/#3-quota-availability).
```

## 4. Rate Limiting

Rate limiting allows you to [limit the amount of events Sentry accepts](/product/accounts/quotas/#what-counts-toward-my-quota-table-view) per project for a defined time span. While this is quite useful for managing your monthly event quota, keep in mind that once a defined threshold is crossed, **subsequent events will be dropped**. Therefore, you shouldn't constantly be hitting your rate limit; rather, it should act as a ceiling intended to protect you from unexpected spikes.

## 5. Delete & Discard Issues {#5-applying-workflows}

> Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, delete and discard it.

Author: ```suggestion
Now that your error events have been fine-tuned to reflect real problems in your code, it's an excellent practice to react to errors as they happen. If an issue reflects a real problem in your code, resolve it; otherwise, delete and discard it. Learn more in [Delete & Discard Issues](/product/accounts/quotas/#5-delete--discard-issues).
```

## 6. Inbound Data Filters

While SDK configuration requires changes to your source code and depends on your next deployment, server-side filters can be easily configured per project in the "Data Filters" section of **[Project] > Settings > Inbound Filters**. Learn more in [Inbound Filters](/product/accounts/quotas/#inbound-data-filters).

Once applied, you can track the filtered events (numbers and cause) using the graph provided at the top of the "Inbound Filters" page.

![Built-in Inbound Filters](manage-event-stream-03.png)

## 7. Event Grouping

Proper event grouping is essential to maintaining a meaningful set of issues and reducing redundant notifications. Sentry groups similar error events into unique _[issues](/product/issues/)_ based on their _fingerprint_. An event's fingerprint relies firstly on its _stack trace_.

## 8. SDK Sample Rate

> blah blah

Author: ```suggestion
blah blah

Learn more in [SDK Sample Rate](/product/accounts/quotas/#8-sdk-sample-rate).
```

## 9. SDK Filtering: beforeSend

All Sentry SDKs support the `beforeSend` callback method. Once implemented, the method is invoked when the SDK captures an event, right before sending it to your Sentry account. It receives the event object as a parameter, so you can use that to modify the event's data or drop it completely (by returning `null`) based on your custom logic and the data available on the event, like _tags_, _environment_, _release version_, _error attributes_, and so on. Learn more in [Filtering Events](/platform-redirect/?next=/configuration/filtering/).

## 10. SDK Configuration

> The Sentry SDKs have several configuration options that can be used to filter unwanted errors from leaving your application's runtime. A lot of these options are platform-specific, so make sure you look for yours in our docs under [Platforms](/platforms/) and [Configuration](/platform-redirect/?next=/configuration/). Following are some examples.

Author: ```suggestion
The Sentry SDKs have several configuration options that can be used to filter unwanted errors from leaving your application's runtime. A lot of these options are platform-specific, so make sure you look for yours in our docs under [Platforms](/platforms/) and [Configuration](/platform-redirect/?next=/configuration/). Learn more in [SDK Configuration](/product/accounts/quotas/#10-sdk-configuration).
```
